charter/index.html

```diff
@@ -163,7 +163,7 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>Cloud services in use on the Web today (2023) often require users to store their data and place control over that data at a third-party cloud provider. Solid adds to existing Web standards to enable user control: <q cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">to realise a space where individuals can maintain their autonomy, control their data and privacy, and choose applications and services to fulfill their needs.</q> Solid defines the notion of storage (one feature of the construct informally referred to as Solid Pods), in which users place their own data and control access to that data, and a suite of interoperable protocols for managing storage, applications that use storage, and interactions with existing protocols for authentication.
+        <p>Cloud services in use on the Web today (2023) often require users to store their data and place control over that data at a third-party cloud provider. Solid adds to existing Web standards to enable user control: <q cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">to realize a space where individuals can maintain their autonomy, control their data and privacy, and choose applications and services to fulfill their needs.</q> Solid defines the notion of storage (one feature of the construct informally referred to as Solid Pods), in which users place their own data and control access to that data, and a suite of interoperable protocols for managing storage, applications that use storage, and interactions with existing protocols for authentication.
```

Comment:
This is a quote. Not sure how technical you'd like but wouldn't the following be more accurate (also noting now that you changed the spelling in an earlier PR):

```suggestion
        <p>Cloud services in use on the Web today (2023) often require users to store their data and place control over that data at a third-party cloud provider. Solid adds to existing Web standards to enable user control: <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">to [<span lang="en-US">realize</span>] a space where individuals can maintain their autonomy, control their data and privacy, and choose applications and services to [<span lang="en-US">fulfill<span>] their needs.</q> Solid defines the notion of storage (one feature of the construct informally referred to as Solid Pods), in which users place their own data and control access to that data, and a suite of interoperable protocols for managing storage, applications that use storage, and interactions with existing protocols for authentication.
```

Reply:
Oh, I hadn't noticed/realized that was a quotation block ... nor any others that may have been. I'm OK with falling back to GB spelling within such. And I should try to remember that Solid seems to be bilingually US/GB and somewhat prefer GB.

Reply:
Sure, that works too. The document is in en-US. Updated suggestion to keep lang en-GB.